<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>你帮我助</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iamges.qrc">
    <normaloff>:/icon.jpg</normaloff>:/icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QStackedWidget#stackedWidget
{
	border-image: url(:/bk.jpg);
	ui-&gt;DayEdit-&gt;setDate(QDate::currentDate());
}
QStackedWidget#stackedWidget QPushButton
{
	border:none;
	border-radius:5px;
	background-color: rgb(0, 102, 227);
	color: rgb(0, 0, 0);
}
QStackedWidget#stackedWidget QPushButton:hover
{
	background-color: rgb(0, 0, 163);
}
QLabel{color: rgb(0, 0, 0);}
QTabWidget#Usertab
{
	background-color: rgb(124, 124, 191);
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>640</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>960</width>
     <height>640</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>960</width>
     <height>640</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="AdminPage">
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>841</width>
       <height>541</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Usertab">
      <attribute name="title">
       <string>用户管理</string>
      </attribute>
      <widget class="QTableWidget" name="userTable">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>771</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="changeauth">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>20</y>
         <width>101</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>权限修改</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>19</y>
         <width>69</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>用户权限</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="authEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>-1新用户   0普通用户  1管理员</string>
       </property>
      </widget>
      <widget class="QPushButton" name="refresh">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>20</y>
         <width>71</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="itemstab">
      <attribute name="title">
       <string>物品信息管理</string>
      </attribute>
      <widget class="QTableWidget" name="itemTable">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>541</width>
         <height>421</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="ExitBtn_3">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>10</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PreviewBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>预览</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="loginPage">
    <widget class="QPushButton" name="adminSigninBtn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>390</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>管理员登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ExitBtn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>470</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>退出系统</string>
     </property>
    </widget>
    <widget class="QLabel" name="Logo">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>521</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>你帮我助物品交换系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="userSigninBtn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>350</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>用户登录</string>
     </property>
    </widget>
    <widget class="QLabel" name="password">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>300</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>密   码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="userNameIn">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>260</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="placeholderText">
      <string>请输入用户名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordIn">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>300</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>请输入密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="userName">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>260</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="RegisterBtn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>430</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>注册用户</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>610</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>作者：艾科Akkk   2022年12月  上海</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="UserPage">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>600</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="SearchBtn">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
    <widget class="QComboBox" name="TypeBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 290 9pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="placeholderText">
      <string>全部</string>
     </property>
     <item>
      <property name="text">
       <string>全部</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>果实</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>饮料</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>零食</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>蔬菜</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>50</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入物品名称</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>611</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="backBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>◀◀返回</string>
     </property>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>你帮我助物品交换系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="ExitBtn_2">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>0</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>类型</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="delBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>570</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>✖删除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changeBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>570</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addBtn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>570</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>➕添加</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NameLin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>请输入物品名称</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EmailLin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>300</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhEmailCharactersOnly</set>
     </property>
     <property name="placeholderText">
      <string>请输入电子邮件地址</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="DayEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2021</year>
       <month>12</month>
       <day>27</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>yyyy/M/d H:mm</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="NumLin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>请输入属性（个数、保质期…）</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="AddressLin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>220</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>请输入物品所在地址</string>
     </property>
    </widget>
    <widget class="QLabel" name="Phonenum">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>联系电话</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Extra">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>备注信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Num">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>物品属性</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Type1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>物品类型</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="PhonenumLin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>11</number>
     </property>
     <property name="placeholderText">
      <string>请输入联系电话</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ExtraLin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>380</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="placeholderText">
      <string>请输入…</string>
     </property>
    </widget>
    <widget class="QLabel" name="Name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>物品名称</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Day">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>添加日期</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Email">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>电子邮件</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Address">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>所在地址</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="TypeBox_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>请选择</string>
     </property>
     <item>
      <property name="text">
       <string>请选择</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>饮料</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>蔬菜</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>水果</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>零食</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="iamges.qrc"/>
 </resources>
 <connections/>
</ui>
